src/cpp/server/auth/ServerAuthHandlerOverlay.cpp

```diff
@@ -125,7 +125,8 @@ Error checkForUninitializedUsername(const boost::shared_ptr<IConnection>& connec
                                     const system::User& user,
                                     std::string* pUsername)
 {
-   // we don't expect the username to be uninitialized in open source
+   // we don't expect the username to be uninitialized in open source, so just set it and return
+   *pUsername = row.get<std::string>(0);
```

Comment:
:0 good catch!

Reply:
😅 😅 😅 

I was testing the companion changes in Pro, and realized I totally missed the assignment here 🤦🏻‍♀️